job_spec/SRG_GSLC.yml

```diff
@@ -61,6 +56,8 @@ SRG_GSLC:
         - Ref::bucket_prefix
         - Ref::granules
       timeout: 10800
+      compute_environment:
+        from_file: SrgGslc
```

Comment:
I might prefer something like `import` instead of `from_file`, just because when I first read `from_file: 'ComputeEnvName1'`, I thought it meant the compute environment was defined in a file named `ComputeEnvName1`.